prisma/seed/forms/checkinform.ts

```diff
@@ -74,12 +74,21 @@ export const populateCheckinForm = async () => {
                                         data: [
                                             {
                                                 text: "{{greenRocket}} We have had a good start!",
+                                                parseConfig: {
+                                                    greenRocket: "🚀",
```

Comment:
I think this needs to be something like 
```js
parseConfig: {
  icon: "greenRocket"
  iconUrl: "....png" // if we want to add this in future, we can, but not required as frontend will match greenRocket with their own icon
}

```

Reply:
Yes, I think that would make it easier to attach an icon or css settings to icon on the frontend.